deployments/staging/00_options.config

```
@@ -8,3 +8,9 @@ option_settings:
     StreamLogs: true
     DeleteOnTerminate: true
     RetentionInDays: 3
+  "aws:elasticbeanstalk:managedactions":
+    ManagedActionsEnabled: true
+    PreferredStartTime: "Mon:00:00"
```

基本的には問題ないと思っているのですが、深夜にする方がいいかと思うので
```suggestion
    PreferredStartTime: "Sun:18:00"
```
はどうでしょうか？